### ISSUE TITLE
NTR: DHPR low in leukocytes (Lab: blood chemistry: enzymes: leukocyte)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8141]
New term request DHPR low in leukocytes (segal_071105195314): Dihydropteridine reductase  UMLS: 2749442

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000966
- label: Diminished tissue dihydropteridine reductase activity
- parents: [Abnormal tissue enzyme concentration or activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:7627180
- def: Concentration or activity of dihydropteridine reductase (EC 1.6.99.7) below the lower limit of normal. This enzyme can be measured in multiple tissues including leukocytes and erythrocytes.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8141
- synonyms:
